.github/workflows/main.yml

```diff
@@ -1,6 +1,10 @@
 name: build
 
-on: [push, pull_request]
+on: 
```

Comment:
Just tidying this up so we don't have duplicate builds when pushing to PR branches not on `main`.